book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] According to your `associations` I think you should update the `add_rental()` method so it has ``person``, ``date`` as parameters, and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`  You take a look at **[this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)** for some examples.
e,g:
```ruby
  def add_rental(person, date) 
    Rental.new(person, self, date)
  end
```